frontend/src/Components/Link/Link.tsx

```diff
@@ -78,7 +78,7 @@ export default function Link<C extends ElementType = 'button'>({
 
   return (
     <Component
-      type={type || 'button'}
+      type={component === 'button' ? type || 'button' : undefined}
```

Comment:
IIRC the `type` was let through even when the component wasn't either a button nor an input

```suggestion
const defaultType =
    component === 'button' || component === 'input' ? 'button' : undefined;

  return (
    <Component
      type={type || defaultType}
```